data/crac-file/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/threshold/AbstractFlowThreshold.java

```diff
@@ -67,14 +67,20 @@ public PhysicalParameter getPhysicalParameter() {
 
     @Override
     public Optional<Double> getMinThreshold(Unit requestedUnit) throws SynchronizationException {
-        // TODO : handle cases where direction != Direction.BOTH
-        return Optional.of(-convert(getAbsoluteMax(), unit, requestedUnit));
+        if (direction == Direction.OUT) {
+            return Optional.of(Double.NEGATIVE_INFINITY);
```

Comment:
We should return Optional.empty() -> It means there is no Threhold limits and in that case there is no constraint in the optimisation problem (and we do not build a dummy constraint Flow >= -infinity).

Reply:
switched to Optional.empty()
(modified a few Double.MIN_VALUE / Double.MAX_VALUE to Double.NEGATIVE_INFINITY / Double.POSITIVE_INFINITY)